datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/SchemaTable.tsx

```diff
@@ -55,7 +55,7 @@ export default function SchemaTable({
     const [selectedFkFieldPath, setSelectedFkFieldPath] =
         useState<null | { fieldPath: string; constraint?: ForeignKeyConstraint | null }>(null);
 
-    const descriptionRender = useDescriptionRenderer(editableSchemaMetadata);
+    const descriptionRender = useDescriptionRenderer(editableSchemaMetadata, editMode);
```

Comment:
i would prefer to pass editMode in via react context so we dont have to pass the param all around. Something like

`SchemaEditableContext`?

Reply:
yes, done